UCSD/campus-mobile
# Cards connectivity

## Summary

[Card Management Online/Offline Status Does Not Change#1749](https://app.zenhub.com/workspaces/campus-mobile-replatform-59791ae0251e9315c45be418/issues/ucsd/campus-mobile/1749)

## Changelog

[General] [Fix] - Added Internet connectivity listener


## Test Plan

1. Go to the Cards page on Profile with Internet connection, then disconnect the Internet. The expected behavior is a popup of no Internet notice.
2. Go to the Cards page on Profile with Internet disconnected, then turn on the connection. The expected behavior is normal cards page show up after hitting close the no Internet popup. 
Refer to the specific behaviors in [the video demo](https://drive.google.com/file/d/1uT-aG872VqxILv4GlAIBWi8wpIrPdJ4-/view?usp=sharing)
